### DISCUSSION TITLE
https://mnfienen.github.io/talulat/2023/08/02/day1.html?fbclid=IwAR0W8aNDnB7sCvbwDAxlZx8iPQNHdINSHf0BkttxEhN2gumM6L1aUZOOcuI_aem_AaNq115tv-2LrKsYqG3KPo5nkdqS-wom0MePx5PrtAAREb2It-NDkJ96yM0mNJO7AwE

### DISCUSSION THREAD

**Discussion by giscus** [mnfienen/talulat#5] [Comments]
# https://mnfienen.github.io/talulat/2023/08/02/day1.html?fbclid=IwAR0W8aNDnB7sCvbwDAxlZx8iPQNHdINSHf0BkttxEhN2gumM6L1aUZOOcuI_aem_AaNq115tv-2LrKsYqG3KPo5nkdqS-wom0MePx5PrtAAREb2It-NDkJ96yM0mNJO7AwE

Day 1 - finally some riding from Sarnia to London and meeting up with good friends! miles today: 76 total miles: 76 km today: 122 total km: 122

https://mnfienen.github.io/talulat/2023/08/02/day1.html?fbclid=IwAR0W8aNDnB7sCvbwDAxlZx8iPQNHdINSHf0BkttxEhN2gumM6L1aUZOOcuI_aem_AaNq115tv-2LrKsYqG3KPo5nkdqS-wom0MePx5PrtAAREb2It-NDkJ96yM0mNJO7AwE

**Comment by SmittysGitHub** [comment]
Some would have been frustrated by flat tires and squeaks.  Not you two.  Love it!
